src/engine/enginedelay.cpp

```diff
@@ -24,12 +24,12 @@
 const int kiMaxDelay = 40000; // 208 ms @ 96 kb/s
```

Comment:
Don't we need to tweak this? Currently we have a 78 kByte buffer. 
We most likely do not need a longer value for all use cases. Can't we pass the buffer size as constructor parameter?